.gitignore

```diff
@@ -3,3 +3,18 @@ __pycache__
 
 # Visual Studio Code
 .vscode
+
+# Ignore python virtual env folders
+.venv
+env
+venv
+
+# Data Folders
+data
+downloads
+
+# Temporary Scrips
+tmp_*.py
+
+# Local Configiuration files
```

Comment:
configuration

Reply:
@yairl 
Making sure I got this one - you prefer to use "configuration" as the folder name and not the "config" shorthand.
Correct?

Reply:
It's a typo - Configiuration.
Just asked to fix it :)

Reply:
🤦 done.